test/VoteManager.js

```diff
@@ -65,13 +65,13 @@ describe('VoteManager', function () {
         );
         const tx = voteManager.connect(signers[5]).commit(epoch, commitment1);
 
-        await assertRevert(tx, 'Contract should be initialized');
+        assertRevert(tx, 'Contract should be initialized');
```

Comment:
Why was await removed?

Reply:
await is no longer required dicussed yesterday @hrishikeshio told to remove await keyword. And it is not making any difference. Also all test cases are passing locally on my terminal. I can't really figure out why are they failing. 
I will Check again.
@SkandaBhat

Reply:
Promises are being handled